dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -15,47 +15,47 @@ properties:
     num-queues:
       type: int
       required: true
-      description: Number of hardware TX and RX queues
+      description: |
+        Number of hardware TX and RX queues.
 
     max-frame-size:
-        type: int
-        description:
-          Maximum transfer unit (IEEE defined MTU), rather than the
-          maximum frame size (there\'s contradiction in the Devicetree
-          Specification). The current supported values are 1518, 1536
-          and 10240 (jumbo frames).
-        default: 1518
+      type: int
+      default: 1518
```

Comment:
Thanks for splitting that up, it's easier to read now.

If you have time, please also add a justification for this default, since you are fixing DT bindings issues. But that was a problem before this PR so I'm not blocking on that request.